src/components/NftCard/NftCard.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import { Meta, Story } from '@storybook/react'
+import React from 'react'
+
+import { NftCard, NftCardProps } from './NftCard'
+
+export default {
+  title: 'other/NFT Card',
+  component: NftCard,
+  argTypes: {
+    title: {
+      type: 'string',
+      defaultValue: 'Did An Alternate Reality Game Gone Wrong Predict QAnon?',
+    },
+    thumbnail: {
+      thumbnailUrl: {
+        type: 'string',
+      },
+    },
+    creator: { table: { disable: true } },
+    supporters: { table: { disable: true } },
+    owner: { table: { disable: true } },
+  },
+  args: {
+    thumbnail: { thumbnailUrl: 'https://placedog.net/360/203' },
+    creator: { assetUrl: 'https://placedog.net/100/100?random=1', tooltipText: 'Jane' },
+    supporters: [
+      { assetUrl: 'https://placedog.net/100/100?random=2', tooltipText: 'Kate' },
+      { assetUrl: 'https://placedog.net/100/100?random=3', tooltipText: 'Sue' },
+      { assetUrl: 'https://placedog.net/100/100?random=4', tooltipText: 'Morgan' },
+      { assetUrl: 'https://placedog.net/100/100?random=5', tooltipText: 'Jeff' },
+      { assetUrl: 'https://placedog.net/100/100?random=6', tooltipText: 'Alan' },
+      { assetUrl: 'https://placedog.net/100/100?random=7', tooltipText: 'Marietta' },
+    ],
+    owner: { assetUrl: 'https://placedog.net/100/100?random=8', tooltipText: 'John' },
+  },
+} as Meta
+
+const Template: Story<NftCardProps> = (args) => (
+  <div style={{ width: '360px' }}>
```

Comment:
shouldn't we apply this width to NFTCard itself?

Reply:
@toiletgranny does Nft Card should have fixed width?

Reply:
<img width="221" alt="Screenshot 2022-01-10 at 15 04 16" src="https://user-images.githubusercontent.com/12646744/148778815-b6a0772d-0aba-40fd-bd4a-15374dba67c5.png">

Seems so

Reply:
@kdembler Exactly! There's nothing inside of the NFT card component that could determine its width. And so the width should be fixed, with an option of being adjusted depending on the use case (eg. on one page we might want to set the width of this component to be of 3 columns, whereas on another page we might want it to span across 2 columns only.)